src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$icons-width: 24px;
+$icons-height: 24px;
+$letter-spacing: -1.5px;
+$navigation-padding-mobile: 40px;
+$navigation-padding-top: 52px;
+$column-gap-mobile: 20px;
+$column-gap: 24px;
+$light-blue: #e8eff9;
```

Comment:
Generally, a good practice is to name variables with colors with descriptive names `$color-light-blue`, `$color-light-black`, `$color-silver`